compiler/record-minmax/include/RecordMinMax.h

```diff
@@ -39,6 +49,9 @@ class RecordMinMax
   void profileData(const std::string &mode, const std::string &input_data_path,
                    float min_percentile, float max_percentile);
 
+  void profileData_with_parallel_record(const std::string &mode, const std::string &input_data_path,
```

Comment:
Please use consistent naming.

```suggestion
  void profileDataInParallel(const std::string &mode, const std::string &input_data_path,
```